server/plugin.go

```diff
@@ -488,7 +488,7 @@ func (p *Plugin) onActivate() error {
 		Displayname:  pluginID,
 		PluginID:     pluginID,
 		CreatorID:    p.userID,
-		AutoShareDMs: true,
+		AutoShareDMs: false,
 		AutoInvited:  true,
```

Comment:
@jespino @lieut-data is we what to ensure no ShareChannel records get created we need to disable AutoInvited as well.